resolwe_bio/processes/icount/xlsites.yml

```diff
@@ -0,0 +1,90 @@
+  description: Quantity and determine positions of cross-link events
+  category: analyses
+  version: 1.0.0
+  type: data:bed:icount
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:icount"
+  data_name: "iCount-xlsites ({{bam.bam.file|basename|default('?')}})"
+  flow_collection: sample
+  persistence: CACHED
+  input:
+  - name: bam
+    label: BAM file with mapped reads
+    type: data:alignment:bam
+  - name: group_by
+    label: Assign score of a read to either 'start', 'middle' or 'end' nucleotide
+    type: basic:string
+    default: start
+  - name: quant
+    label: Report number of 'cDNA' or number of 'reads'
+    type: basic:string
+    default: cDNA
+  - name: segmentation
+    label: File with custon annotation format (obtained by iCount segment)
+    type: data:annotation:gtf:icount
+    required: false
+  - name: mismatches
+    label: >
+        Reads on same position with random barcode differing less than
+        mismatches are grouped together.
+    type: basic:integer
+    default: 2
+  - name: mapq_th
+    label: Ignore hits with MAPQ < mapq_th
+    type: basic:integer
+    default: 0
+  - name: multimax
+    label: Ignore reads, mapped to more than ``multimax`` places
+    type: basic:integer
+    default: 50
+  - name: gap_th
+    label: Reads with gaps less than gap_th are treated as if they have no gap
+    type: basic:integer
+    default: 4
+  - name: report_progress
+    label: Switch to report progress
+    type: basic:boolean
+    default: false
+  output:
+  - name: BED
+    label: Output BED6 file to store data from uniquely mapped reads
+    type: basic:file
+  - name: BED_multi
+    label: Output BED6 file to store data from multi-mapped reads
+    type: basic:file
+  - name: skipped
+    label: >
+        BAM file with reads that do not map as expected by annotation and
+        reference genome sequence. If read's second start does not fall on any
+        of annotation borders, it is considered problematic. If segmentation is
+        not provided, every read in two parts with gap longer than gap_th is not
+        used (skipped). All such reads are reported to the user for further
+        exploration.
+    type: basic:file
+  run:
+    language: bash
+    runtime: polyglot
+    program: |
+      iCount xlsites {{bam.bam.file}} unique.bed.gz multi.bed.gz skipped.bam \
+          --group_by {{group_by}} \
+          --quant {{quant}} \
+          {% if segmentation %} --segmentation {{segmentation}}{% endif %} \
+          --mismatches {{mismatches}} \
+          --mapq_th {{mapq_th}} \
+          --multimax {{multimax}} \
+          --gap_th {{gap_th}} \
+          {% if report_progress %} --report_progress {% endif %}
+
+      re-checkrc
+
+      re-progress 0.9
+      re-save-file BED unique.bed.gz
+      re-save-file BED_multi multi.bed.gz
+      [[ -e ./skipped.bam ]] && echo "skipped.bam was produced" || touch skipped.bam
```

Comment:
You can make `skipped` output field as required: false, check if the file was produced and save it if it actually was.